Dockerfile

```diff
@@ -125,6 +125,8 @@ RUN if [[ -n $TIMESTAMP ]]; then \
 
 RUN \
     if [[ -n $ITB ]]; then \
+        # Set the default pool to ITB, but allow turning off with -e ITBPOOL=0
+        echo 'export ITBPOOL=${ITBPOOL:-1}' > /etc/osg/image-init.d/01-itb.sh; \
```

Comment:
Any reason you're doing this instead of just setting `ENV ITBPOOL=-1` in the Dockerfile?

Reply:
I only want to set the default of ITBPOOL if the `ITB` build arg is set, and Docker doesn't really have `if` statements.